blocks/brightcove/brightcove.js

```diff
@@ -0,0 +1,38 @@
+import { readBlockConfig } from '../../scripts/aem.js';
+
+export default async function decorate(block) {
+  const config = readBlockConfig(block);
+
+  const picture = block.querySelector('picture');
```

Comment:
I think this is an optimisation which might not be needed. I would go with IntesectionObserver approach (as we have done in https://github.com/hlxsites/servicenow/pull/63)